README.md

```diff
@@ -1,36 +1,78 @@
-Piper
+Piper [![CircleCI](https://circleci.com/gh/libratbag/piper.svg?style=svg&circle-token=7082ad7a7fea706fff66f1547649dca32e446cb0)](https://circleci.com/gh/libratbag/piper)
```

Comment:
This is probably a good idea to have the badge here.

Please rebase on top of master to remove the ugly "Build Status" entry I added.